guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+
+> If thieves can see your income, holdings, and spending, they can use this information to target and exploit you.[^2]
+
+Using the same address multiple times does not only degrade the privacy of just one transacting party. Both the [sender](/guide/payments/receive) and [receiver](/guide/payments/receive) are put at risk. On the receiver’s side, by using a single address, senders or anyone with whom the address is shared can see the previous activity and the amount of bitcoin controlled by the owner.
+
+Assuming an adversary uncovers the identity behind the reused address there is now a single point for them to reach anyone else who has transacted with the owner.
+
+<!--
+TODO: Link / mention gap limit
+TODO: Graphic / consider how to get the ui generating multiple addresses. make it easy for the user to not fail 
+
+> ###### Worked Example 1 - Savings Revealed
+> * You save in bitcoin, using a single-address paper wallet.
+> * All your bitcoin savings to this same address, let's say it contains $1 million worth.
+> * You buy a small amount of bitcoins to add to your savings, depositing in the paper wallet.
+> * The person who sold you the bitcoins follows their trail on the blockchain and finds your paper wallet containing $1 million.
+> * He mentions it to someone in a cafe or bar.
+> * Word gets around. A burglar raids your home. Kidnappers capture your children and know exactly how much to demand in ransom.
+
+-->
+
+
+**Do's**
+
+- Show a new unused address when the receive screen is entered.
+- Make it easy to generate as many addresses as the receiver needs at a time.
+- Before broadcasting a transaction, inform senders if an address is already used.
+
+**Don'ts**
+
+- Make it easy to reuse an address.
+
+### Keep track of who knows about an address
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/address-labeling.jpg"
+   retina = "/assets/images/guide/payments/privacy/address-labeling@2x.jpg"
+   alt-text = "A list of addresses with tags. Two are red, two are purple and one is blue"
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+When [receiving bitcoin]({{ "/guide/payments/receive/" | relative_url }}) we can [add additional details to a payment]({{ "/guide/payments/receive/#inputting-additional-payment-details" | relative_url }}). Not only does this help to contextualize payments, but it also enables preventative measures for preserving privacy. Labeling addresses with the counterparty's name involved in each transaction can inform decisions for which [UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }}) are selected in future transaction inputs.
+
+To make this UTXO selection easier, some interfaces make it possible to select/filter UTXOs that cluster under one label.
```

Comment:
Could add an image here showcasing bluewallets UTXO management? 

After talking last night I think a page dedicated to UTXO management is probably needed in the payments section...

Reply:
UTXO page should be this PR https://github.com/BitcoinDesign/Guide/pull/90